README.md

```diff
@@ -47,15 +47,7 @@ considerably. For example, to build a metadata index for Fenix (Firefox for
 Android) only, try:
 
 ```bash
-./scripts/build-glean-metadata fenix
-```
-
-For the search service _only_ (see below), the Glean Dictionary also indexes
-legacy Firefox telemetry metadata. To build this index (not necessary for most
-testing), you need to run a seperate script:
-
-```bash
-./scripts/build-legacy-metadata
+./scripts/build-metadata fenix
```

Comment:
Hmmm this command doesn't look right. Did you mean `./scripts/gd build-metadata fenix`?

Reply:
Yes! Oops